types.ts

```diff
@@ -0,0 +1,16 @@
+export type testResults = {
```

Comment:
Should we make it singular? `testResult`?

Reply:
Yes. I think just for now 'cause you could need to modificate the types for your use case to use on the submission page or with puppteer lib